lib/open_project/costs/engine.rb

```diff
@@ -65,11 +65,11 @@ class Engine < ::Rails::Engine
       end
 
       # Menu extensions
-      menu :top_menu,
+      menu :admin_menu,
            :cost_types,
            {:controller => '/cost_types', :action => 'index'},
-           :caption => :cost_types_title,
-           :if => Proc.new { User.current.admin? }
+           :html => { :class => 'icon2 icon-tracker' },
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.